dotCMS/src/main/java/com/dotcms/util/user/LiferayUserTransformer.java

```diff
@@ -0,0 +1,76 @@
+package com.dotcms.util.user;
+
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.liferay.portal.ejb.UserPool;
+import com.liferay.portal.model.User;
+import java.util.Map;
+
+/**
+ * This class implement the UserTransformer interface
+ * to convert dotconnect maps from user to User
+ */
+public class LiferayUserTransformer implements UserTransformer {
+
+
+    @Override
+    public User fromMap(Map<String, Object> map) throws DotStateException {
+        User user = null;
+        if(null != map) {
+            user = new com.liferay.portal.model.User();
+            user.setUserId((String) map.get("userid"));
+            user.setCompanyId((String) map.get("companyid"));
+            user.setCreateDate((java.util.Date) map.get("createdate"));
+            user.setModificationDate((java.util.Date) map.get("mod_date"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "mod_date" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)